app/views/orders/_order_notes_table.html.erb

```diff
@@ -0,0 +1,22 @@
+<table <%= 'style="display: none;"'.html_safe if local_assigns[:hidden] %> class="table table-hover" id="order-notes-table">
+  <thead>
+    <tr>
+      <th>Note</th>
+      <th>Actions</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% order_notes.each do |order_note| %>
+      <tr>
+        <td>
+          <%= order_note.text %>
```

Comment:
Are you familiar with [simple_format](https://apidock.com/rails/ActionView/Helpers/TextHelper/simple_format)? I think it could be useful here to provide a small amount of formatting to the notes without much complexity.